server/service/src/sync/test/integration/transfer/shipments.rs

```diff
@@ -0,0 +1,310 @@
+use repository::ItemRow;
+use serde_json::json;
+use util::{inline_init, uuid::uuid};
+
+use crate::{
+    processors::transfer::shipment::test::ShipmentTransferTester,
+    sync::test::integration::transfer::{new_instance_of_existing_site, sync_and_delay},
+};
+
+use super::{initialise_transfer_sites, SyncIntegrationTransferContext};
+
+#[actix_rt::test]
+async fn integration_sync_shipment_transfers_normal() {
+    // util::init_logger(util::LogLevel::Info);
+
+    let item1 = inline_init(|r: &mut ItemRow| {
+        r.id = uuid();
+    });
+
+    let item2 = inline_init(|r: &mut ItemRow| {
+        r.id = uuid();
+    });
+
+    let SyncIntegrationTransferContext {
+        site_1,
+        site_2,
+        site_1_processors_task,
+        site_2_processors_task,
+    } = initialise_transfer_sites(
+        json!({
+            "item": [
+                {"ID": item1.id, "type_of": "general"},
+                {"ID": item2.id, "type_of": "general"},
+            ]
+        }),
+        "sync_shipment_transfers_normal",
+    )
+    .await;
+
+    let test = async move {
+        let mut tester = ShipmentTransferTester::new(&site_1.store, &site_2.store, &item1, &item2);
+
+        log::info!("Inserting request requisition on site {:?}", site_2.config);
+        tester
+            .insert_request_requisition(&site_2.service_provider)
```

Comment:
In requisition tests site 1 creates the requisition_request. Is this deliberately the other way around?
It does seem logical to test both directions, however it feels is slightly harder to reason about for me (probably because I'm coming straight from reading the other test?)

Reply:
Hmm, it's because I added requisitions to shipment tests after shipment tests were done. And at that point the first site that edit records was site1 (the supplying site). But yeah it's a little bit weird, should I change them over ? Btw this diagram kind of shows it quite well: https://github.com/openmsupply/open-msupply/blob/584-Transfer-integration-tests/server/service/src/processors/transfer/doc/omSupply_transfer_workflow.png

Reply:
I have an idea, instead of site1 or site2, they should be named `request_site`, `response_site` and `inbound_and_request_site` and `outbound_and_response_site`

Reply:
This is much better i think: 67872a1ae32ae55d32254f44ab7db567cfb7cf30